changelog/3975.trivial.rst

```diff
@@ -0,0 +1 @@
+Remove legacy code around im_func as that was python2 only
```

Comment:
Python 2.6 you mean?

Reply:
@nicoddemus python2.7 supports the modern properties but was also based on the old ones, i believe the message is accurate enough ^^

Reply:
Oh OK, now I get it. Thanks!